README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
+
+    export MYSQL_USER=<YOUR_USERNAME>
+    export MYSQL_PASSWORD=<YOUR_PASSWORD>
+    export MYSQL_HOST=<PASTE_YOUR_HOST_HERE>
+    export MYSQL_PORT=<3306>
+    export DATABASE_NAME=msia423asg5718
+
+
+Set the environment variables in your ~/.bashrc
+
+    echo 'source .mysqlconfig' >> ~/.bashrc
+
+    source ~/.bashrc
+    
+PLEASE NOTE: VERIFY THAT YOU ARE ON THE NORTHWESTERN VPN BEFORE YOU CONTINUE ON.        
+
+* SET AWS_ACCESS_KEY with your aws access key.
+* SET AWS_SECRET_KEY with you aws secret key.
+
+Run the following commands and replace xxxx with keys etc.
+
+    nano ~/.bashrc
+Note you may find 'source .mysqlconfig' already, leave it untouched.
+
+    export AWS_ACCESS_KEY=xxxx
+    export AWS_SECRET_KEY=xxxxx
+    source .mysqlconfig
+Don't forget to source:
 
-#### Create the database with a single song 
-To create the database in the location configured in `config.py` with one initial song, run: 
+    source ~/.bashrc
+Create environments variables for docker:
 
-`python run.py create_db --engine_string=<engine_string> --artist=<ARTIST> --title=<TITLE> --album=<ALBUM>`
+    cd config
+Open config.env file
 
-By default, `python run.py create_db` creates a database at `sqlite:///data/tracks.db` with the initial song *Radar* by Britney spears. 
-#### Adding additional songs 
-To add an additional song:
+    nano config.env
+Set credentials - replace the xxx here too :
 
-`python run.py ingest --engine_string=<engine_string> --artist=<ARTIST> --title=<TITLE> --album=<ALBUM>`
+    AWS_SECRET_KEY=xxxxxxxxxxxxxxxxx
+    AWS_ACCESS_KEY=xxxxxxxxxx
 
-By default, `python run.py ingest` adds *Minor Cause* by Emancipator to the SQLite database located in `sqlite:///data/tracks.db`.
+
+### 4.Set configurations:
+Go to config.py in the config directory:
+
+    cd config/ 
+    
+ Within `config.py` Please change the following bucket name with your own bucket name.: 
+ 
+    S3_BUCKET_NAME = 'nw-adityagudal-s3'
+    
+To create local SQLlite schema let the variable `local` be as is, currently it is:
+    
+    local=True
+ 
+To create RDS instance change the variable `local`:
+     
+     local=False
+
+### 5. Store data into S3
+
+First check step 3 and 4, make sure your:
+   * Environment variables are set.
+   * Requirements and dependencies are met.
+   * Set configurations in config.py
+   
+Set your bucket name in `config.py` within config directory.
+
+Go to the root directory and Run the following:
+
+  * Build a docker image with tag games:
+  
+  		docker build -t games .
+
+   * Run the docker container:
+   
+    docker run --env-file=config/config.env games run.py
+
+    
+Note this `run.py` depends on `src/store_data_in_s3.py`
+
+This should store your data into the S3 bucket.
+
+### 6. Initialize the database 
+
+#### Create the database with tables
+ First check step 3 and 4, make sure your:
+   * Environment variables are set.
+   * Requirements and dependencies are met.
```

Comment:
good job on checking in with the user to make sure he/she has completed all previous steps.